java/connector-node/risingwave-connector-service/src/main/java/com/risingwave/connector/source/core/DbzChangeEventConsumer.java

```diff
@@ -168,14 +181,66 @@ var record = event.value();
                                         record.topic(), record.valueSchema(), record.value());
                         var message =
                                 msgBuilder
-                                        .setIsTransactionMeta(true)
+                                        .setMsgType(CdcMessage.CdcMessageType.TRANSACTION_META)
                                         .setPayload(new String(payload, StandardCharsets.UTF_8))
                                         .setSourceTsMs(trxTs)
                                         .build();
                         LOG.debug("transaction => {}", message);
                         respBuilder.addEvents(message);
                         break;
                     }
+
+                case SCHEMA_CHANGE:
+                    {
+                        var ddl = ((Struct) record.value()).getString("ddl");
+                        if (ddl.contains("CREATE") || ddl.contains("DROP")) {
```

Comment:
This can be prone to bug. If the table name for ALTER contains `CREATE` / `DROP`, we will incorrectly skip the schema change event. Example: `ALTER TABLE CREATE_TIME ADD COLUMN ...`

Reply:
We should check the `type` under `tableChanges` instead.